src/System.Net.WebSockets.WebSocketProtocol/src/System/Net/WebSockets/ManagedWebSocket.netstandard.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+// NOTE: This file is shared between CoreFX and ASP.NET.  Be very thoughtful when changing it.
```

Comment:
This comment and the others like it can be removed.  The purpose of this change is to remove the need for such code sharing.